### PR TITLE
doc: migration-guide: annotate various entry with the PR numbers

### PR DESCRIPTION
**Pull request by fabiobaltieri**
Add a PR number reference to the input touchscreen polarity change entry.

### CODE REVIEW

**Comment by henrikbrixandersen** [comment]
We should probably tag all entries like this in one go.

**Comment by fabiobaltieri** [comment]
> We should probably tag all entries like this in one go.

Yeah, was mainly wanting to use this as an excuse to discuss the format.

**Review by kartben — DISMISSED** [review]

**Review by carlescufi — DISMISSED** [review]

**Comment by fabiobaltieri** [comment]
Did the other ones as well

File: doc/releases/migration-guide-3.6.rst [hunk 1]
```diff
@@ -30,7 +30,7 @@ anymore:
 
 To enable them again use the ``west config manifest.project-filter -- +<module
 name>`` command, or ``west config manifest.group-filter -- +optional`` to
-enable all optional modules, and then run ``west update`` again.
+enable all optional modules, and then run ``west update`` again. (:github:`64139`)
```
Comment (henrikbrixandersen) [inline doc/releases/migration-guide-3.6.rst:33]: This reference should go at the end of the line on the canopennode" bullet instead.

**Review by henrikbrixandersen — CHANGES_REQUESTED** [review]
Thanks, one minor change.

File: doc/releases/migration-guide-3.6.rst (hunk 1 above)
Comment (fabiobaltieri) [reply doc/releases/migration-guide-3.6.rst]: updated, is that what you meant?

**Review by henrikbrixandersen — APPROVED** [review]
Thanks!

File: doc/releases/migration-guide-3.6.rst (hunk 1 above)
Comment (henrikbrixandersen) [reply doc/releases/migration-guide-3.6.rst]: Exactly, thanks.

**Comment by fabiobaltieri** [comment]
https://builds.zephyrproject.io/zephyr/pr/65313/docs/releases/migration-guide-3.6.html preview link (still waiting...)

**Review by kartben — APPROVED** [review]
Thanks a bunch, Fabio!

**Comment by thedjnK** [comment]
Not really so useful for modules this, the MCUboot one links to a zephyr module update which pulls in multiple commits, would be better to link to the module PR which made the change and has a description https://github.com/zephyrproject-rtos/mcuboot/commit/4a1effbc301fc302ecbaf40d4eb2be520b53010d

**Comment by fabiobaltieri** [comment]
> Not really so useful for modules this, the MCUboot one links to a zephyr module update which pulls in multiple commits, would be better to link to the module PR which made the change and has a description [zephyrproject-rtos/mcuboot@4a1effb](https://github.com/zephyrproject-rtos/mcuboot/commit/4a1effbc301fc302ecbaf40d4eb2be520b53010d)

I think normally it would be useful for module, as the module PR is going to contain the upstream changes as well to make the existing code work with the module, which I think is the value of this (for example finding how samples and board have been modified). It's more like: not so useful for deprecated options that have been deleted... what do you expect to find in the PR? :-)

Plus the github tag does not support modules -- it'd be nice if it did though I'll see if I can make that work but that'll prob need its own PR, unless you feel strongly about it I'd say let's merge this and we can update it on a followup.

**Comment by henrikbrixandersen** [comment]
> Plus the github tag does not support modules -- it'd be nice if it did though I'll see if I can make that work but that'll prob need its own PR, unless you feel strongly about it I'd say let's merge this and we can update it on a followup.

I think the reference from the migration guide should continue to point to the Zephyr repo PR. All needed information should be accessible from there.

**Review by carlescufi — APPROVED** [review]
Thanks!!!

**Pull request merged**